<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>944</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>944</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ruta Corta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tw_Contenedor">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>4</y>
      <width>931</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Multigrafo">
     <attribute name="title">
      <string>Multigrafo</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>621</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="layoutGrafo"/>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>654</x>
        <y>6</y>
        <width>120</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Filtro</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="cb_Dias">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_Hora">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>74</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>46</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dia</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>79</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Hora</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>166</y>
        <width>160</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Calcular Ruta</string>
      </property>
      <widget class="QLineEdit" name="le_Origen">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>21</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_Destino">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>71</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>71</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destino</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Origen</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Consultar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>140</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="te_RutaOptima">
      <property name="geometry">
       <rect>
        <x>648</x>
        <y>318</y>
        <width>271</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Aristas</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDia">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>40</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Grafo para dia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbHora">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hora:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Floy">
     <attribute name="title">
      <string>Floy</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_floy"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>160</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="idem"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionCargar_Grafo"/>
    <addaction name="actionAgregar_Arista"/>
    <addaction name="actionRemover_Arista"/>
   </widget>
   <addaction name="menu_Archivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCargar_Grafo">
   <property name="text">
    <string>Cargar Grafo</string>
   </property>
   <property name="toolTip">
    <string>Carge un grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAgregar_Arista">
   <property name="text">
    <string>Agregar o Modificar Arista</string>
   </property>
   <property name="toolTip">
    <string>Agregar Arista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemover_Arista">
   <property name="text">
    <string>Remover Arista</string>
   </property>
   <property name="toolTip">
    <string>Borra Arista Entre 2 Nodos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
